app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PostsController < ApplicationController
```

Comment:
- As mentioned by the previous reviewer, Kindly add corresponding view files for the index and show controller actions, for your posts and user controller 👍🏿. I noticed you added only index.html.erb for user. Please add the show HTML.erb for users and the both index.html.erb and show.html.erb for post.